lumigator/python/mzai/schemas/schemas/jobs.py

```diff
@@ -50,16 +50,31 @@ class JobSubmissionResponse(BaseModel):
     driver_exit_code: int | None = None
 
 
-class JobCreate(BaseModel):
+class JobEvalCreate(BaseModel):
```

Comment:
I'd advice against this split. I'd rather make something like this instead:

```
class JobCreate(BaseModel):
    name: str
    description: str = ""
    model: str
    dataset: UUID
    max_samples: int | None = None
    model_url: str | None = None
    system_prompt: str | None = None
    job_specific_config: BaseModel | None = None
```

So each job can pass its own config without further specification, and when using model_validate in the backend each job can check their own config with their own model.

Reply:
Just seeing this after proposing something - [wdyt of this](https://github.com/mozilla-ai/lumigator/pull/328#discussion_r1826153789)?